maintainers/maintainer-list.nix

```diff
@@ -8444,6 +8444,12 @@
     githubId = 3747396;
     name = "Nathan Isom";
   };
+  NelsonJeppesen = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add NAME`` and sort it alphabetically.